### PR TITLE
Create model

### PR DESCRIPTION
**Pull request by at-cuclt**
@at-vienlm please help me review this.

### CODE REVIEW

File: app/AdminUser.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    protected $fillable = ['username', 'password', 'fullname', 'created_at', 'updated_at'];
+
+    protected $hidden = [
+        'password'
+    ];
+
+    /**
+     * Get all category for AdminUser .
+     *
+     * @return array
```
Comment (ghost) [inline app/AdminUser.php]: not return array, please check the Laravel core to get the return class. And please fix for all. Thanks

File: app/AdminUser.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    protected $hidden = [
+        'password'
+    ];
+
+    /**
+     * Get all category for AdminUser .
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function cate()
```
Comment (ghost) [inline app/AdminUser.php]: `category` is a better name than `cate`

File: app/Book.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        return $this->belongsTo('App/AdminUser', 'admin_user_id');
+    }
+
+    /**
+     * Get all bookItem for book.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function bookItem()
```
Comment (ghost) [inline app/Book.php]: `bookItems` is better

File: app/BookItem.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        return $this->belongsTo('App/Book', 'book_id');
+    }
+
+    /**
+     * Get the borrowDetail from BookItem.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function borrowDetail()
```
Comment (ghost) [inline app/BookItem.php]: `borrowDetails` is better

File: app/AdminUser.php (hunk 2 above)
Comment (at-cuclt) [inline app/AdminUser.php]: yes, thanks.

File: app/Borrow.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        return $this->belongsTo('App/AdminUser', 'admin_user_id');
+    }
+
+    /**
+     * Get borrowDetail from Borrow.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function borrowDetail()
```
Comment (ghost) [inline app/Borrow.php]: `borrowDetails` is better

File: app/Category.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    public function adminUser()
+    {
+        return $this->belongsTo('App/AdminUser', 'admin_user_id');
+    }
+    /**
+     * Get book from Category.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function book()
```
Comment (ghost) [inline app/Category.php]: `books` is better

File: app/User.php [hunk 7]
```diff
@@ -23,4 +25,14 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    /**
+     * Get borrow from User.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function borrow()
```
Comment (ghost) [inline app/User.php]: `borrows` is better

**Comment by ghost** [comment]
Please follow these rules:
- HasMany, BelongsToMany -> function name should be in plural
- HasOne, BelongsTo -> function name should be in single
  Thanks

File: app/AdminUser.php [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+class AdminUser extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $table = 'admin_users';
+
+    protected $fillable = ['username', 'password', 'fullname', 'created_at', 'updated_at'];
```
Comment (ghost) [inline app/AdminUser.php]: `created_at`, `updated_at` should not in `$fillable`

**Pull request merged**